packages/gatsby-plugin-netlify/src/build-headers-program.js

```diff
@@ -228,7 +228,7 @@ const writeHeadersFile = ({ publicFolder }) => contents =>
 export default function buildHeadersProgram(pluginData, pluginOptions) {
   return _.flow(
     validateUserOptions(pluginOptions),
-    mapUserLinkHeaders(pluginData, pluginOptions),
+    mapUserLinkHeaders(pluginData),
```

Comment:
This function takes three parameters.

Reply:
it's one parameter but it gets destructured 
```
{
  manifest,
  pathPrefix,
  publicFolder,
}
```